.github/workflows/linting.yml

```
@@ -6,15 +6,22 @@ on:
 
 jobs:
   asciidoc:
-    name: 'Build AsciiDocs'
+    name: Build AsciiDocs
     runs-on: ubuntu-latest
     steps:
       - name: Check out the repository
-        uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579
+        uses: actions/checkout@v4
```

Tags can be moved, which causes silent updates to actions; it’s best to refer to specific commits (and ideally, review the changes in actions). dependabot knows how to handle this.

```suggestion
        uses: actions/checkout@44c2b7a8a4ea60a981eaca3cf939b5f4305c123b
```